cli/ops/fs.rs

```diff
@@ -30,80 +30,88 @@ use deno_core::error::generic_error;
 #[cfg(not(unix))]
 use deno_core::error::not_supported;
 
-pub fn init(rt: &mut deno_core::JsRuntime) {
-  super::reg_json_sync(rt, "op_open_sync", op_open_sync);
-  super::reg_json_async(rt, "op_open_async", op_open_async);
+pub fn init(state: &mut deno_core::OpState) {
+  super::reg_json_sync(state, "op_open_sync", op_open_sync);
+  super::reg_json_async(state, "op_open_async", op_open_async);
```

Comment:
These changes seem superfluous - what is the purpose?

 I like the init functions to take a `&mut JsRuntime` for maximum extensibility.

Reply:
I'm not so happy about this change either, however when I tried adding state elements next to call to `init()` I've hit `BorrowMutError` numerous times - because `rt.register_op()` mutably borrow `OpState`. It's quite cumbersome to register state that way without this change...

Reply:
I'll give it another go